src/routes.json

```diff
@@ -19,6 +19,20 @@
       "online": true,
       "offline": true
     },
+    {
+      "name": "patient-diagnoses",
+      "slot": "extra-patient-information-slot",
```

Comment:
These extensions should _not_ be bound to any slot by default. That way, they are already able to be enabled or disabled by configuration.